dgraph/cmd/zero/http.go

```diff
@@ -231,9 +234,49 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) zeroHealth(ctx context.Context) (*api.Response, error) {
+	if ctx.Err() != nil {
+		return nil, ctx.Err()
+	}
+
+	healthAll := []pb.HealthInfo{pb.HealthInfo{
+		Instance:    "zero",
+		Address:     x.WorkerConfig.MyAddr,
+		Status:      "healthy",
+		Group:       strconv.Itoa(int(worker.GroupId())),
+		Version:     x.Version(),
+		Uptime:      int64(time.Since(x.WorkerConfig.StartTime) / time.Second),
+		LastEcho:    time.Now().Unix(),
+	}}
+
+	jsonOut, _ := json.Marshal(healthAll)
```

Comment:
we should check the error here, than ignoring it.

Reply:
I misunderstood your earlier comment. This is corrected in the recent upload.